examples/pytorch/AxHyperOptimizationPTL/iris.py

```diff
@@ -34,8 +34,7 @@ def __init__(self, **kwargs):
     def forward(self, x):
         x = F.relu(self.fc1(x))
         x = F.relu(self.fc2(x))
-        x = F.relu(self.fc3(x))
-        return x
+        return F.relu(self.fc3(x))
 
```

Comment:
I hardly saw ppl doing this, rather than vertically aligning layers like the original one...

Reply:
Ok, then let's revert them :)

Reply:
@B-Step62 reverted